src/xdp-utils.c

```diff
@@ -884,6 +884,36 @@ verify_proc_self_fd (XdpAppInfo *app_info,
   return xdp_app_info_remap_path (app_info, path_buffer);
 }
 
+static char *documents_mountpoint = NULL;
+
+void
+xdp_set_documents_mountpoint (const char *path)
+{
+  g_clear_pointer (&documents_mountpoint, g_free);
+  documents_mountpoint = g_strdup (path);
+}
+
+
+/* alternate_document_path converts a file path  */
+char *
+alternate_document_path(const char *path, const char *app_id)
```

Comment:
Might be useful to have a simple unit test to test this codepath. Don't forget to add a space before the parenthesis.

Reply:
I've added a test to `test-xdp-utils.c`.